t/lib/t/MusicBrainz/Server/Controller/Artist/Filtering.pm

```diff
@@ -633,6 +633,30 @@ test 'Work page filtering' => sub {
         'The entry is named "Interlude"',
     );
 
+    $mech->get_ok(
+        '/artist/af4c43d3-c0e0-421e-ac64-000329af0435/works?filter.language_id=120',
+        'Fetched artist works page with language filter "English"',
+    );
+
+    $tx = test_xpath_html($mech->content);
+    $tx->is(
+        'count(//table[@class="tbl"]/tbody/tr)',
+        '2',
+        'There are two entries in the work table after filtering by "English" language',
+    );
+
+    $mech->get_ok(
+        '/artist/af4c43d3-c0e0-421e-ac64-000329af0435/works?filter.language_id=-1',
+        'Fetched artist works page with language filter "[none]"',
+    );
+
+    $tx = test_xpath_html($mech->content);
+    $tx->is(
+        'count(//table[@class="tbl"]/tbody/tr)',
+        '3',
+        'There are three entries in the work table after filtering by no language',
```

Comment:
To follow https://github.com/metabrainz/musicbrainz-server/compare/d2a068fe1ebe859f01df695a5bc4bcf91286d311..727c65fab53f2759573bf8d29c82fdbc8541ce0f

```suggestion
        'There are three entries in the work table after filtering by no language set',
```